frontend/javascripts/oxalis/geometries/helper_geometries.ts

```diff
@@ -231,3 +314,113 @@ export class QuickSelectGeometry {
     rectangle.material.alphaMap = null;
   }
 }
+
+export class LineMeasurementGeometry {
+  color: THREE.Color;
```

Comment:
maybe add a class comment explaining that this is used for displaying "line segments" (not sure whether this is the right term?).

Reply:
I called it `connected line segments` :shrug: